listener.c

```diff
@@ -228,8 +232,15 @@ evconnlistener_new_bind(struct event_base *base, evconnlistener_cb cb,
 	if (fd == -1)
 		return NULL;
 
-	if (setsockopt(fd, SOL_SOCKET, SO_KEEPALIVE, (void*)&on, sizeof(on))<0)
-		goto err;
+#if defined(_WIN32) && defined(EVENT__HAVE_AFUNIX_H)
```

Comment:
I guess this is here because windows do not support SO_KEEPALIVE for non AF_UNIX, while linux does ([simply do not report any error](https://elixir.bootlin.com/linux/v5.3.6/source/net/core/sock.c#L850)), so comment will be nice